### DISCUSSION TITLE
Writing single band image in FITS format

### DISCUSSION THREAD

**Discussion by ewelot** [libvips/libvips#2085] [Ideas]
Currently the function fitssave() is writing a FITS-cube (NAXIS=3) regardless of the number of existing bands. I'd suggest writing single band images to a simple FITS plane (NAXIS=2) instead for better compatibility with several other programs processing FITS files (e.g. the widely used [astromatic](http://www.astromatic.net/) software).

**Comment by jcupitt** [comment]
Good idea, let's tag this as an enhancement.
